strings/src/main/java/com/revenuecat/purchases/strings/NetworkStrings.kt

```diff
@@ -0,0 +1,6 @@
+package com.revenuecat.purchases.strings
+
+object NetworkStrings {
+    const val HTTP_REQUEST_COMPLETED = "API request completed with status: %s %s %s"
+    const val HTTP_REQUEST_STARTED = "API request started: %s %s"
```

Comment:
these should be `API_REQUEST_COMPLETED` and `API_REQUEST_STARTED`, otherwise if we make an http request to something that isn't our API, we might reasonably be tempted to use this string